### PR TITLE
Leaflet wayfinding example

### PR DESCRIPTION
**Pull request by oseiskar**
Based on branch `sdk-29-test-app` (see [diff](https://github.com/IndoorAtlas/sdk-cordova-examples/compare/sdk-29-test-app...new-leaflet-wayfinding)), replaces old `leaflet-wayfinding` branch. The current LeafletExample does not use wayfinding and uses old APIs

### CODE REVIEW

File: LeafletExample/README.md [hunk 1]
```diff
@@ -23,13 +23,15 @@ and Android
 cordova platform add android
 ```
 
-Add IndoorAtlas Cordova plugin
-```
-cordova plugin add https://github.com/IndoorAtlas/cordova-plugin.git
```
Comment (simomar) [inline LeafletExample/README.md]: Why was this removed?

Comment (oseiskar) [reply LeafletExample/README.md]: I tried following the instructions on a new project and it appears this is not really needed since the plugin is already defined in the `config.xml`
